packages/manager/apps/dedicated/client/app/billing/payment/method/add/controller.js

```diff
@@ -99,6 +108,17 @@ export default class BillingPaymentMethodAddCtrl {
   /* ----------  OuiStepper callbacks  ---------- */
 
   onPaymentMethodAddStepperFinish() {
+    if (this.model.selectedPaymentMethodType.isHandleByComponent()) {
+      this.componentInitialParams = this.onPaymentMethodIntegrationInitialized(
+        null,
+        {
+          locale: Environment.getUser().language,
```

Comment:
this part will be refactored after merge of this PR : https://github.com/ovh/manager/pull/4487